cli/admin/README.md

```diff
@@ -52,8 +52,8 @@ You can run the Pravega Admin CLI as follows:
 ```
 ./bin/pravega-admin
 ```
-You will se an output related to the default configuration parameters available at `conf/config.properties`
-(you may want to change this file according to your setting):
+The config values can be changed permanently at `conf/admin-cli.properties`, or use `config set property=...` to temporarily change the config for command line session only.
+The initial configuration:
```

Comment:
Something is missing in this sentence. Please, rephrase it like: `The initial configuration would be as follows:` or similar.